package.json

```diff
@@ -2,13 +2,25 @@
   "name": "branch-manager",
   "version": "0.0.0-beta.1",
   "private": true,
+  "engines": {
+    "node": "8.x",
+    "yarn": ">=1.0.0 < 2.0.0"
+  },
   "scripts": {
+    "start": "node ./dist/src/presubmit-service/index.js",
     "lint": "tslint --project tsconfig.json",
-    "build": "tsc -b tsconfig.json"
+    "build": "rm -rf dist/ && tsc -b tsconfig.json",
+    "serve-github-event": "yarn build && functions deploy githubEvents --trigger-http",
+    "serve-task": "yarn build && functions deploy handlePresubmitPrTask --trigger-topic presubmit-pr-task",
+    "serve-presubmit": "yarn build && cp ./src/presubmit-service/check_if_pr_can_cherry_pick.sh ./dist/src/presubmit-service/ && node ./dist/src/presubmit-service/index.js",
+    "deploy-github-event": "yarn build && gcloud functions deploy githubEvents --trigger-http --runtime nodejs8",
+    "deploy-task": "yarn build && gcloud functions deploy handlePresubmitPrTask --trigger-topic presubmit-pr-task --runtime nodejs8",
+    "deploy-presubmit": "yarn build && cp ./src/presubmit-service/app.yaml . && cp ./src/presubmit-service/check_if_pr_can_cherry_pick.sh ./dist/src/presubmit-service/ && gcloud app deploy --no-promote && rm app.yaml"
```

Comment:
Could you add a `README` that describes what each of these tasks do? Preferably in table form.